### PR TITLE
create ProfileMedia container

### PR DESCRIPTION
**Pull request by Tretmax**
Снимок

Create Profile media container with infinite list

### CODE REVIEW

**Comment by Mykyta-Pysarev** [comment]
Shouldn`t you add this layout to media tab in profile?

File: src/containers/ProfileMedia.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      docs: [...(mockMediaPosts?.docs || []), ...mockData],
+      hasNextPage: true,
+      limit: 5,
+      page: page + 1,
+    });
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
```
Comment (alexandrtovmach) [inline src/containers/ProfileMedia.tsx]: remove everything from 13 to 31 lines

File: src/containers/ProfileMedia.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  return (
+    <>
+      {mockMediaPosts && (
+        <InfiniteList
+          showMore={fetchData}
```
Comment (alexandrtovmach) [inline src/containers/ProfileMedia.tsx]:
```suggestion
          showMore={() => {}}
```

File: src/containers/ProfileMedia.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+
+import SingleTweet, { SingleTweetInterface } from "../components/SingleTweet";
+import dataProfileMediaContainer from "../MOCKS/dataProfileMediaContainer.json";
+import { PaginationParamsResult } from "../types/mock-api-types";
+import InfiniteList from "./InfiniteList";
+
+function sleep(ms: number) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
```
Comment (alexandrtovmach) [inline src/containers/ProfileMedia.tsx]: remove everything from 8 to 10 lines

File: src/containers/ProfileMedia.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  return (
+    <>
+      {mockMediaPosts && (
+        <InfiniteList
+          showMore={fetchData}
+          data={mockMediaPosts}
```
Comment (alexandrtovmach) [inline src/containers/ProfileMedia.tsx]:
```suggestion
          data={dataProfileMediaContainer}
```

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**